.github/workflows/checks.yml

```diff
@@ -15,7 +15,7 @@ jobs:
                   node-version: 20
                   cache: 'npm'
             - run: npm run install-ci
-            - run: npm run lint
```

Comment:
I would prefer not to have it automatically run `lint-fix`, but I think having the guidance here is nice. My vote would be to tweak this to just explain to the user to try running `npm run lint-fix`. WDYT @sammacbeth ?

Reply:
_(Thinking about it, if we _don't_ go with my suggestion above perhaps we should even just combine the `lint` and `lint-fix` commands. Otherwise, we'd have two commands that do basically the same thing.)_

Reply:
@kzar note that lint-fix is executed only to generate and print the diff in CI logs. It doesn't actually commit anything and the CI check will still fail

Reply:
I think this is fine as-is: this will only run lint-fix if there is a failure to show a better error message in the logs.